### PR TITLE
add config examples

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/roadmap/issues/3521

Add Alloy configuration examples for scrapping logs using PodLogs and Loki and importing PrometheusLogs into Mimir.

### CODE REVIEW

**Review by marieroque — APPROVED** [review]

File: helm/alloy/examples/logs/podlogs.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: monitoring.grafana.com/v1alpha2
```
Comment (QuentinBisson) [inline helm/alloy/examples/logs/podlogs.yaml:1]: Did you explain how to déploy the CRDs?

File: helm/alloy/examples/mimir-rules/values.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+alloy:
+  alloy:
```
Comment (QuentinBisson) [inline helm/alloy/examples/mimir-rules/values.yaml:2]: I think it would bé cool to add an example with évents as well

**Review by QuentinBisson — APPROVED** [review]

File: helm/alloy/examples/logs/podlogs.yaml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/alloy/examples/logs/podlogs.yaml:1]: They come with Alloy deploment

File: helm/alloy/examples/mimir-rules/values.yaml (hunk 2 above)
Comment (TheoBrigitte) [reply helm/alloy/examples/mimir-rules/values.yaml:2]: We can add this later on, examples with host files would be nice too, like audit logs.

**Pull request merged**